app/frontend/src/search/ui/input.js

```diff
@@ -0,0 +1,16 @@
+import { updateUrlQueryParams } from '../utils'
+
+export const renderSearchBox = (renderOptions, isFirstRender) => {
+    const { query, refine, clear, isSearchStalled, widgetParams } = renderOptions;
+
+    if (isFirstRender) {
+        widgetParams.container.querySelector(widgetParams.element).addEventListener('input', event => {
+            refine(event.target.value);
+        });
+    }
+
+    query ? updateUrlQueryParams(widgetParams.key, query) : false
+    document.querySelector('ul.vacancies').style.display = query ? 'none' : 'block';
+    document.querySelector('ul.pagination-server').style.display = query ? 'none' : 'block';
+    // widgetParams.container.querySelector('span').hidden = !isSearchStalled;
```

Comment:
Please clear commented code before release if it's already in history.

Reply:
Probably worth getting a linter in here sooner rather than later...

Reply:
yep agreed, i wanted to get something visible to people that all. i dont even know what this line does tbh (that should be smell to get rid!) but was a hangover from the algolia docs

Reply:
i have this in linting now, might a few commented out bits still however that ill certainly remove today